gateway/utils/admin/admin.go

```diff
@@ -58,3 +58,20 @@ func (m *Manager) Login(user, pass string) (int, string, error) {
 
 	return http.StatusUnauthorized, "", errors.New("invalid credentials provided")
 }
+// RefreshToken is used to create a new token!
+func (m *Manager) RefreshToken(token string) (int,string,error){
+	m.lock.RLock()
+	defer m.lock.RUnlock()
+	// Parse the token to get userID and userRole
+	tokenClaims ,err := m.parseToken(token)
+	if err != nil {
```

Comment:
In this pr wherever you have encountered an error, log the error using logrus.Errorf
for e.g -> for the following function
` logrus.Errorf("error while refreshing token in admin unable to parse token - %s ",err.Error())`